packages/syft/src/syft/client/domain_client.py

```diff
@@ -182,6 +182,7 @@ def upload_model(self, model: CreateModel) -> SyftSuccess | SyftError:
             return valid
 
         # Step 4. Upload Model to Model Stash
+        model.asset_list = []
```

Comment:
@rasswanth-s the changes in `on_disk.py` alone reduces the `upload_model` time from ~1m20s to ~30s. Adding this line further reduces the total time to ~5s.

From what I gather (and I might be wrong here), we are already uploading these assets to the action store in step 2. Can we have some sort of lazy loading for this asset list instead, where they contain only the action ids, and the actual objects can be fetched on demand instead?

Reply:
This is because of the data and mock field in the asset, we could delete them after uploading to blob storage.
I will push a commit fixing the same.

Reply:
Done [2358ede](https://github.com/OpenMined/PySyft/pull/9053/commits/2358ede3116ac276d17b1ff62bcacd72b20160bc)